viewer/packages/camera-manager/src/ComboControls.ts

```diff
@@ -711,20 +709,25 @@ export default class ComboControls extends EventDispatcher {
     deltaTargetOffsetDistance *= deltaDownscaleCoefficient;
 
     let radius = distToTarget + deltaDistance;
-
+    
+    //console.log(radius, minDistance, deltaDistance, distToTarget, _target);
+    //console.log(_camera.position);
     // behaviour for scrolling with mouse wheel
     if (radius < minDistance) {
       this._temporarilyDisableDamping = true;
 
       // stops camera from moving forward only if target became close to scroll target
-      if (_scrollTarget.distanceTo(_target) < minDistance) {
-        radius = minDistance;
+      if ((_scrollTarget.distanceTo(_target) < minDistance) || (radius <= 0)) {
         deltaTargetOffsetDistance = 0;
+        
+        radius = distToTarget;
```

Comment:
Shouldn't radius be `radius = Math.max(distToTarget, minDistance);` ?

Reply:
In that case it would lead to the same problem of slight movement in cases when it equals minDistance. Because when you change the radius to some new value you also need to offset the target accordingly (for correct zooming movement). That part of calculating proper target offset was the initial problem I was struggling with.